sdk/python/requirements/py3.10-requirements.txt

```diff
@@ -20,32 +20,38 @@ charset-normalizer==3.3.2
     # via requests
 click==8.1.7
     # via
+    #   feast (setup.py)
```

Comment:
Did you use `lock-python-dependencies-all` to generate these filese? `feast (setup.py)` lines shouldn't have been added, I think.

Reply:
Yes, I used that command indeed!

Reply:
Not sure honestly, I'll try to look into it. We can still merge regardless, it's not a blocker, just a bunch of extra line changes in the PR.